sensor/DNSCryptPlugin.js

```diff
@@ -100,8 +101,10 @@ class DNSCryptPlugin extends Sensor {
     }
   }
 
-  async applyAll() {
-    await dc.prepareConfig({});
+  async applyAll(reCheckConfig = false) {
+    log.info("DNSCryptPlugin.applyAll");
+    const result = await dc.prepareConfig({}, reCheckConfig);
+    if (!result) return;
```

Comment:
assume there is a possibility that the dnscrypt service crashed, but the file content is still there nicely.

The current code will not restart the service.

better to call `sudo systemctl start dnscrypt` to ensure it's started up